test/parser/samples/error-event-handler/error.json

```diff
@@ -1,5 +1,5 @@
 {
-	"message": "Expected call expression",
+	"message": "Expected CallExpression",
```

Comment:
I think this is *less* expressive than using English.

Reply:
I agree, it is more literal. Are you thinking we should write a method to humanize the expression type in the errors?